slave/cracker.go

```diff
@@ -0,0 +1,31 @@
+package slave
+
+import (
+	b64 "encoding/base64"
+	"fmt"
+
+	"github.com/aaronang/cong-the-ripper/lib"
+	"github.com/aaronang/cong-the-ripper/slave/brutedict"
+	"github.com/aaronang/cong-the-ripper/slave/hasher"
+)
+
+func Execute(task lib.Task, slave *Slave) {
+	bd := brutedict.New(&task)
+	digest, _ := b64.StdEncoding.DecodeString(task.Digest)
```

Comment:
The reason that I chose string is that go will base64 encode the []byte content. So the digest in the http request would then be an base64-encoded base64-encoded digest (if you know what I mean :P)